src/assets/config-template.json

```diff
@@ -4,7 +4,8 @@
   ],
   "distributeStationsToTenantsEqually": true,
   "performanceStorage": {
-    "enabled": true
+    "enabled": true,
+    "type": "jsonfile"
```

Comment:
The default value in `Configuration.ts` if empty or not present is already set to `jsonfile`. Do you have a configuration file that trigger that default to not be taken into account?

Reply:
I have debuged it. That check controls if there is performanceStorage property. 

if yes, it assigns the values inside of that property. However by default in the config-template there is performanceStorage property with only enabled parameter value.

So, in the code, it assigns default value "jsonfile" to "type" parameter, then in the if-clause, since there is no type property, it is removed from storageConfiguration object. this results in "undefined" type and the code breaks. I will push another commit for this

Reply:
Ok, I let you handle the proper fix in `Configuration.ts` then.